devices/GnssDevice/GnssDevice.cs

```diff
@@ -77,29 +127,219 @@ protected set
         /// <summary>
         /// Gets or sets the last known location.
         /// </summary>
-        public GeoPosition Location
+        public Location Location
         {
             get => _location;
             protected set
             {
+                if (_location == value)
+                {
+                    return;
+                }
+
                 _location = value;
                 LocationChanged?.Invoke(_location);
             }
         }
 
         /// <summary>
-        /// Represents the event handler for when the fix status of the Gnss module changes.
+        /// Gets or sets the satellites in view.
```

Comment:
The description seems to differ from the property name... should be either in view or in use... don't know the details, so not sure which one is correct.